src/libstd/lib.rs

```diff
@@ -268,9 +268,13 @@ pub mod sync;
 pub mod time;
 
 #[macro_use]
+#[cfg_attr(target_os = "nacl", allow(dead_code))]
+#[cfg_attr(target_os = "nacl", allow(unused_imports))]
 #[path = "sys/common/mod.rs"] mod sys_common;
 
 #[cfg(unix)]
+#[cfg_attr(target_os = "nacl", allow(dead_code))]
+#[cfg_attr(target_os = "nacl", allow(unused_imports))]
```

Comment:
When I added those two lines, it was either bite the bullet or coat the module in a glaze of `#[cfg]`s. Granted, that decision was made back when I disabled `libstd`s ability to spawn them and was done by making the relevant functions always return `Err`. This left a pile of unused helper functions and imports.

I've since removed that restriction (so I could use `rustc` inside PNaCl), so I'll see if it's still a problem.